libs/core/src/lib/list/list-item/list-item.component.ts

```diff
@@ -109,15 +103,15 @@ export class ListItemComponent
     @HostBinding('class.fd-list__item--unread')
     unread = false;
 
-    /** Text to be read by screen reader for selected list item */
+    /** @deprecated Text to be read by screen reader for selected list item */
```

Comment:
This is damage control for backwards compatibility. Typically like to leave inputs in place, with the deprecated tag, for a few release cycles since they will break templates immediately

Reply:
Yes, but when deprecated property is used usually it has the same effect as not deprecated. In our case, property is deprecated but in reality it is removed because it is not used anywhere. We should either deprecate and keep the functionality or we should remove it and provide ng-update script so that users have it removed automagically